tests/unit/data/test_extract_metadata.py

```diff
@@ -35,16 +35,16 @@ def cube_1():
     @staticmethod
     @pytest.fixture
     def cube_2():
-        latitude = DimCoord(np.linspace(1, 100, 100),
+        latitude = DimCoord(np.linspace(1, 100, 200),
                             standard_name='projection_x_coordinate',
                             units='meters')
-        longitude = DimCoord(np.linspace(1, 100, 100),
+        longitude = DimCoord(np.linspace(1, 100, 200),
```

Comment:
This seems fine because longitude can fall between 0-360 (when offset, with an offset it's -180 to 180).
Just ensure you pick an approach (i.e. offset or not) and stick with it to avoid confusion (unless you're explicitly testing that code can handle the different styles)